tests/bazel_tests.bzl

```diff
@@ -1,8 +1,13 @@
 _bazelrc = """
-build --fetch=False --verbose_failures --sandbox_debug --test_output=errors --spawn_strategy=standalone --genrule_strategy=standalone
+startup --batch
+build --verbose_failures --sandbox_debug --test_output=errors --spawn_strategy=standalone --genrule_strategy=standalone
```

Comment:
It will make for easier future diffs to have one option per line, either as:
build \
  --verbose_failures \
  --sandbox_debug \
  ...
or as:
build --verbose_failures
build --sandbox_debug
...